pkgs/applications/science/electronics/verilator/default.nix

```
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Fast and robust (System)Verilog simulator/compiler";
     homepage    = "https://www.veripool.org/wiki/verilator";
-    license     = licenses.lgpl3;
+    license     = licenses.lgpl3Only;
```

The code seems to be [dual-licensed](https://github.com/verilator/verilator#open-license).

```suggestion
    license     = with licenses; [ lgpl3Only artistic2 ];
```